react/components/SearchBar/SearchBar.tsx

```diff
@@ -236,17 +285,20 @@ const SearchBar = ({
                   placeholder,
                   value: inputValue,
                   onChange: onInputChange,
+                  // @ts-expect-error this exists
```

Comment:
We gotta update the preset for this repo

Reply:
Yes, we can do that after we merge all the migration of TS PRs